utils/python/CIME/build.py

```diff
@@ -94,10 +93,11 @@ def build_model(case, build_threaded, exeroot, clm_config_opts, incroot, complis
     file_build = os.path.join(exeroot, "%s.bldlog.%s" % (cime_model, lid))
 
     config_dir = os.path.join(cimeroot, "driver_cpl", "cime_config")
-    stat = run_cmd("%s/buildexe %s %s %s >> %s 2>&1" %
-                   (config_dir, caseroot, bldroot, libroot, file_build),
-                   from_dir=bldroot, ok_to_fail=True, verbose=True)[0]
-
+    f = open(file_build, "w")
```

Comment:
This changes the way that build output is logged, streaming it to the file instead of buffering it in memory.